blocks/v2-footer/v2-footer.js

```diff
@@ -32,9 +32,14 @@ function goToTopFunction() {
 }
 
 function addScrollToTopButton(mainEl) {
-  const scrollToTopButton = createElement('button', ['scroll-to-top', 'button'], { title: 'Go to the top of the page' });
-  const icon = createElement('i', ['fa', 'fa-angle-up']);
-  scrollToTopButton.appendChild(icon);
+  const scrollToTopButton = createElement('button', ['v2-scroll-to-top'], { title: 'Go to the top of the page' });
+  // const icon = createElement('i', ['fa', 'fa-angle-up']);
+  // scrollToTopButton.appendChild(icon);
+  const svgIcon = document.createRange().createContextualFragment(`
+    <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Could we use the SVG "arrow top", instead of arrow right and rotate it with CSS.